### DISCUSSION TITLE
目前酷我的cookie有效期不到一天，不知道是不是我个例

### DISCUSSION THREAD

**Discussion by chris7395328** [1715173329/UnblockNeteaseMusic#1199] [General]
最近版本更新后用自己的cookie正常用了几天，昨天失效后重新区获取，发现hm变成了两行，换上了还是能用，但是今天下午就又不行了，感觉这个频率有点难顶了

**Comment by laggardkernel** [comment]
~~Secret 是什么，该去哪里获得？~~ 找到了，api 接口请求的 header.

**Comment by Inkarro** [comment]
#1207 试着修了下，我给搜索的API换成了手机客户端的，手机客户端的没校验
